assets/ca/ca.querytranslator.js

```diff
@@ -181,8 +192,8 @@ var caUI = caUI || {};
 					queryArray.shift();
 				}
 			} else {
-				// In plain word mode, the next non-word, non-. character ends the token.
-				if (/[\w\.]/.test(character)) {
+				// In plain word mode, the next non-word, non-dot character ends the token.
+				if (/[\w.]/.test(character)) {
```

Comment:
Sideline change -- the `.` character here does not need to be escaped, because you never use the "match-all" type of `.` in a character group, because that would be meaningless ("match any character or these specific characters...").  Not sure why I escaped it in the first place.